src/components/Game.js

```diff
@@ -27,6 +29,22 @@ const Game = () => {
     return setStatusText(`Next player: ${xIsNext ? playerXName : playerOName}`);
   }, [history, playerX, playerO]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const showRules = () => {
+    setIsShowingGameRules(!isShowingGameRules);
+  };
+
+  const hideRules = () => {
+    setIsShowingGameRules(false);
+  };
+
+  const resetGame = () => {
+    setStepNumber(0);
+    setXIsNext(true);
+    setHistory([{ squares: Array(9).fill(null) }]);
+    setStatusText("");
```

Comment:
The `useEffect` hook should handle updating this state to the proper value, no?

Reply:
You are correct. I did not realize that it would reset the state.